src/index.html

```diff
@@ -4,9 +4,356 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;0,900;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="container">
+        <div class="logo">
+          <a href="#" class="logo-link">
+            <img src="/src/images/logo.png" alt="header-logo">
+          </a>
+        </div>
```

Comment:
```suggestion
        <a href="#" class="logo logo-link">
          <img src="/src/images/logo.png" alt="header-logo">
        </a>
```
Why do you need this div container? If you only have one tag, it doesn't make sense to wrap it in a container. This applies to all similar cases in your code.

Reply:
isn't fixed